compiler/test/input/recursive-equal-box.gr

```diff
@@ -44,3 +44,5 @@ b.r := Some(b)
 
 assert !(a is b)
 assert a == b
+
+print(void)
```

Comment:
Why have this test print anything?

Reply:
All of our non-stdlib tests have to print _something_ to check that the test passed with the expected result (but the stdlib tests are just configured to have a zero exit code). I didn't really want to set up a brand new test runner just for these two tests, so this maintained the current test expectation (since before the module was expected to evaluate to `void`). I'll at least change these to `print("OK")` so it's more obvious what's going on.